### DISCUSSION TITLE
Does the documentation site feels laggy and clunky? :/

### DISCUSSION THREAD

**Discussion by AsuraKev** [ant-design/ant-design#50436] [General]
Hi people, does anyone feel that the documentation website feel laggy and not smooth to browse around?

**Comment by zombieJ** [comment]
antd site use babel for compatible with the browser not support latest es standard which cause bundle size & network cost. I want to remove this compatible config for next major version. It may help.

**Reply by AsuraKev** [reply]
Thanks for the quick reply. Do we anticipate a rough time of when v6 might be coming? And potentially leak some changelog that v6 might include?

thanks 🙂
